spec/factories/schedulable_cutoff_dates.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :schedulable_cutoff_date do
+    id { 1 }
+    created_at { Time.zone.now }
+    cutoff_date { Time.zone.today + 30.days }
+    created_by_id do
+      create(:appeal)
+      appeal.id
+    end
```

Comment:
```suggestion
    created_by_id { create(:user).id }
```
I think something like this would allow for the factory to be run on its own without params.